src/racing.domain/CarRacingGame.js

```diff
@@ -0,0 +1,58 @@
+const RACING_GAME = require("../racing.const/game");
+const Car = require("./Car");
+const Generator = require("./GeneratorNumber");
+
+class CarRacingGame {
+  #racers = [];
+  #count = 0;
+  #gameStatus = RACING_GAME.STATUS.PLAY;
+
+  initCarNameList(listStr) {
+    const carList = listStr.split(",");
+    carList.map((racerInfo) => {
+      const racer = new Car(racerInfo);
+      this.#racers.push(racer);
+    });
+  }
+
+  carRacigStart(gamecount) {
+    let result = [];
+    if (this.#count < gamecount) {
+      result = this.racerMoveOnePosition(this.#racers);
+      this.#count += 1;
+    }
+    if (this.#count === gamecount) this.#gameStatus = RACING_GAME.STATUS.END;
+    return result;
+  }
+
+  racerMoveOnePosition(racers) {
+    let result = [];
```

Comment:
Airbnb 스타일 가이드의 참조에 대한 문서에서 문맥을 살펴보시면,
2.1 모든 참조에는 const를 사용하라.
2.2 재할당이 필요한 경우에, let을 사용하라.

그만큼 기본적인 선언에는 재할당이 불가능하도록 const 부터 선언하도록 하고 있어요.
물론 let으로 선언하신 것은 실수이셨겠지만,
const 부터 선언하여 재할당이 필요할 때에 let으로 바꾸는 습관이 실수를 줄일 수 있을 것 같다고 생각해요.

[2.1 Use const for all of your references; avoid using var.](https://github.com/airbnb/javascript#references--prefer-const)
[2.2 If you must reassign references, use let instead of var.](https://github.com/airbnb/javascript#references--disallow-var)